docs/docs/contributing/release.sh

```diff
@@ -0,0 +1,63 @@
+#!/usr/bin/env bash
```

Comment:
Use `set -ex` to stop at the first error and to print every line of execution
```suggestion
#!/usr/bin/env bash
set -ex
```